packages/react-dom/src/__tests__/ReactDOMRoot-test.js

```diff
@@ -175,27 +175,6 @@ describe('ReactDOMRoot', () => {
     );
   });
 
-  it('clears existing children with legacy API', async () => {
```

Comment:
I also moved any legacy tests in this file (which @eps1lon found), and moved them into `ReactLegacyRoot`.